04/09/21 (01:37:54) | ERROR: Exception while getting cpu frequency!
04/09/21 (01:37:54) | ERROR: [Errno 2] No such file or directory (originated from sysctlbyname('hw.cpufrequency'))
Traceback (most recent call last):
  File "/opt/homebrew/bin/bpytop", line 3060, in _collect
    if CONFIG.show_cpu_freq and hasattr(psutil.cpu_freq(), "current"):
  File "/opt/homebrew/Cellar/bpytop/1.0.67_1/libexec/lib/python3.9/site-packages/psutil/__init__.py", line 1859, in cpu_freq
    ret = _psplatform.cpu_freq()
  File "/opt/homebrew/Cellar/bpytop/1.0.67_1/libexec/lib/python3.9/site-packages/psutil/_psosx.py", line 180, in cpu_freq
    curr, min_, max_ = cext.cpu_freq()
FileNotFoundError: [Errno 2] No such file or directory (originated from sysctlbyname('hw.cpufrequency'))
04/09/21 (18:02:47) | ERROR: Exception while getting cpu frequency!
04/09/21 (18:02:47) | ERROR: [Errno 2] No such file or directory (originated from sysctlbyname('hw.cpufrequency'))
Traceback (most recent call last):
  File "/opt/homebrew/bin/bpytop", line 3060, in _collect
    if CONFIG.show_cpu_freq and hasattr(psutil.cpu_freq(), "current"):
  File "/opt/homebrew/Cellar/bpytop/1.0.67_1/libexec/lib/python3.9/site-packages/psutil/__init__.py", line 1859, in cpu_freq
    ret = _psplatform.cpu_freq()
  File "/opt/homebrew/Cellar/bpytop/1.0.67_1/libexec/lib/python3.9/site-packages/psutil/_psosx.py", line 180, in cpu_freq
    curr, min_, max_ = cext.cpu_freq()
FileNotFoundError: [Errno 2] No such file or directory (originated from sysctlbyname('hw.cpufrequency'))
04/09/21 (18:03:04) | ERROR: Exception while getting cpu frequency!
04/09/21 (18:03:04) | ERROR: [Errno 2] No such file or directory (originated from sysctlbyname('hw.cpufrequency'))
Traceback (most recent call last):
  File "/opt/homebrew/bin/bpytop", line 3060, in _collect
    if CONFIG.show_cpu_freq and hasattr(psutil.cpu_freq(), "current"):
  File "/opt/homebrew/Cellar/bpytop/1.0.67_1/libexec/lib/python3.9/site-packages/psutil/__init__.py", line 1859, in cpu_freq
    ret = _psplatform.cpu_freq()
  File "/opt/homebrew/Cellar/bpytop/1.0.67_1/libexec/lib/python3.9/site-packages/psutil/_psosx.py", line 180, in cpu_freq
    curr, min_, max_ = cext.cpu_freq()
FileNotFoundError: [Errno 2] No such file or directory (originated from sysctlbyname('hw.cpufrequency'))
04/09/21 (18:14:07) | ERROR: Exception while getting cpu frequency!
04/09/21 (18:14:07) | ERROR: [Errno 2] No such file or directory (originated from sysctlbyname('hw.cpufrequency'))
Traceback (most recent call last):
  File "/opt/homebrew/bin/bpytop", line 3060, in _collect
    if CONFIG.show_cpu_freq and hasattr(psutil.cpu_freq(), "current"):
  File "/opt/homebrew/Cellar/bpytop/1.0.67_1/libexec/lib/python3.9/site-packages/psutil/__init__.py", line 1859, in cpu_freq
    ret = _psplatform.cpu_freq()
  File "/opt/homebrew/Cellar/bpytop/1.0.67_1/libexec/lib/python3.9/site-packages/psutil/_psosx.py", line 180, in cpu_freq
    curr, min_, max_ = cext.cpu_freq()
FileNotFoundError: [Errno 2] No such file or directory (originated from sysctlbyname('hw.cpufrequency'))
04/09/21 (18:16:58) | ERROR: Exception while getting cpu frequency!
04/09/21 (18:16:58) | ERROR: [Errno 2] No such file or directory (originated from sysctlbyname('hw.cpufrequency'))
Traceback (most recent call last):
  File "/opt/homebrew/bin/bpytop", line 3060, in _collect
    if CONFIG.show_cpu_freq and hasattr(psutil.cpu_freq(), "current"):
  File "/opt/homebrew/Cellar/bpytop/1.0.67_1/libexec/lib/python3.9/site-packages/psutil/__init__.py", line 1859, in cpu_freq
    ret = _psplatform.cpu_freq()
  File "/opt/homebrew/Cellar/bpytop/1.0.67_1/libexec/lib/python3.9/site-packages/psutil/_psosx.py", line 180, in cpu_freq
    curr, min_, max_ = cext.cpu_freq()
FileNotFoundError: [Errno 2] No such file or directory (originated from sysctlbyname('hw.cpufrequency'))
04/09/21 (18:24:34) | ERROR: Exception while getting cpu frequency!
04/09/21 (18:24:34) | ERROR: [Errno 2] No such file or directory (originated from sysctlbyname('hw.cpufrequency'))
Traceback (most recent call last):
  File "/opt/homebrew/bin/bpytop", line 3060, in _collect
    if CONFIG.show_cpu_freq and hasattr(psutil.cpu_freq(), "current"):
  File "/opt/homebrew/Cellar/bpytop/1.0.67_1/libexec/lib/python3.9/site-packages/psutil/__init__.py", line 1859, in cpu_freq
    ret = _psplatform.cpu_freq()
  File "/opt/homebrew/Cellar/bpytop/1.0.67_1/libexec/lib/python3.9/site-packages/psutil/_psosx.py", line 180, in cpu_freq
    curr, min_, max_ = cext.cpu_freq()
FileNotFoundError: [Errno 2] No such file or directory (originated from sysctlbyname('hw.cpufrequency'))
04/09/21 (18:25:31) | ERROR: Exception while getting cpu frequency!
04/09/21 (18:25:31) | ERROR: [Errno 2] No such file or directory (originated from sysctlbyname('hw.cpufrequency'))
Traceback (most recent call last):
  File "/opt/homebrew/bin/bpytop", line 3060, in _collect
    if CONFIG.show_cpu_freq and hasattr(psutil.cpu_freq(), "current"):
  File "/opt/homebrew/Cellar/bpytop/1.0.67_1/libexec/lib/python3.9/site-packages/psutil/__init__.py", line 1859, in cpu_freq
    ret = _psplatform.cpu_freq()
  File "/opt/homebrew/Cellar/bpytop/1.0.67_1/libexec/lib/python3.9/site-packages/psutil/_psosx.py", line 180, in cpu_freq
    curr, min_, max_ = cext.cpu_freq()
FileNotFoundError: [Errno 2] No such file or directory (originated from sysctlbyname('hw.cpufrequency'))
04/09/21 (18:25:40) | ERROR: Exception while getting cpu frequency!
04/09/21 (18:25:40) | ERROR: [Errno 2] No such file or directory (originated from sysctlbyname('hw.cpufrequency'))
Traceback (most recent call last):
  File "/opt/homebrew/bin/bpytop", line 3060, in _collect
    if CONFIG.show_cpu_freq and hasattr(psutil.cpu_freq(), "current"):
  File "/opt/homebrew/Cellar/bpytop/1.0.67_1/libexec/lib/python3.9/site-packages/psutil/__init__.py", line 1859, in cpu_freq
    ret = _psplatform.cpu_freq()
  File "/opt/homebrew/Cellar/bpytop/1.0.67_1/libexec/lib/python3.9/site-packages/psutil/_psosx.py", line 180, in cpu_freq
    curr, min_, max_ = cext.cpu_freq()
FileNotFoundError: [Errno 2] No such file or directory (originated from sysctlbyname('hw.cpufrequency'))
04/09/21 (19:27:03) | ERROR: Exception while getting cpu frequency!
04/09/21 (19:27:03) | ERROR: [Errno 2] No such file or directory (originated from sysctlbyname('hw.cpufrequency'))
Traceback (most recent call last):
  File "/opt/homebrew/bin/bpytop", line 3060, in _collect
    if CONFIG.show_cpu_freq and hasattr(psutil.cpu_freq(), "current"):
  File "/opt/homebrew/Cellar/bpytop/1.0.67_1/libexec/lib/python3.9/site-packages/psutil/__init__.py", line 1859, in cpu_freq
    ret = _psplatform.cpu_freq()
  File "/opt/homebrew/Cellar/bpytop/1.0.67_1/libexec/lib/python3.9/site-packages/psutil/_psosx.py", line 180, in cpu_freq
    curr, min_, max_ = cext.cpu_freq()
FileNotFoundError: [Errno 2] No such file or directory (originated from sysctlbyname('hw.cpufrequency'))
